core/src/main/java/org/apache/iceberg/rest/HTTPClient.java

```diff
@@ -78,6 +80,8 @@ public class HTTPClient implements RESTClient {
   private static final int REST_MAX_CONNECTIONS_DEFAULT = 100;
   private static final String REST_MAX_CONNECTIONS_PER_ROUTE = "rest.client.connections-per-route";
   private static final int REST_MAX_CONNECTIONS_PER_ROUTE_DEFAULT = 100;
+  @VisibleForTesting static final String REST_CONNECTION_TIMEOUT = "rest.client.connection-timeout";
+  @VisibleForTesting static final String REST_SOCKET_TIMEOUT = "rest.client.socket-timeout";
```

Comment:
are those set in millis or in seconds? In other places we always indicate in the property naming whether it's millis or not -> `rest.client.socket-timeout-ms`

Reply:
Yeah, these are seconds. I'll fix the naming.